api/v1beta1/types.go

```diff
@@ -49,6 +49,15 @@ type Install struct {
 	DisableBootEntry bool `json:"disable-boot-entry,omitempty" yaml:"disable-boot-entry,omitempty"`
 	// +optional
 	ConfigDir string `json:"config-dir,omitempty" yaml:"config-dir,omitempty"`
+	// +optional
+	Snapshotter SnapshotterConfig `json:"snapshotter,omitempty" yaml:"snapshotter,omitempty"`
+}
+
+type SnapshotterConfig struct {
+	// Type sets the snapshotter type a new installation, available options are 'loopdevice' and 'btrfs'
+	// +optional
+	// +kubebuilder:default:=loopdevice
```

Comment:
Does this actually work when you don't specify the `snapshotter` field?
In my experience it will create an empty object. 
This is the reason I had to set the defaults at a "parent" level for [Reset](https://github.com/rancher/elemental-operator/blob/main/api/v1beta1/types.go#L111).

Reply:
Indeed, you are right, looks like this is not enough. Testing to set it at upper level too.